docker/Makefile

```diff
@@ -80,6 +80,27 @@ deploy-dev:
 	$(dc) stop app && $(dc) up -d app
 
 
+deploy-stage:
+	# - build section
+	# preserve the dev image from repository
+	$(d) tag fidals/stb:dev fidals/stb:dev_temp
```

Comment:
Why do we need to preserve dev image? What do you mean by this?

Reply:
@artemiy312 , comment will be too big and complex. I'll update docker-compose-build file instead